dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md

```diff
@@ -0,0 +1,625 @@
+---
+slug: static-rating-API
+title: Rating API static documentation
+authors: []
+tags: [rating, api, introduction]
+---
+
+# Overview 
+This document is intended for technical architects and developers that need to integrate with ShipperHQ’s Rating API to build their own integration to retrieve shipping rates from ShipperHQ.
+
+Use the information outlined in this document to view how custom fields and attributes are queried and modified.
+
+Note, this does not document the standard types and fields the API provides. This information is found in the GraphQL schema and obtained via our GraphQL playground. Please refer to the Introduction to GraphQL article from Graphql.org for more information about GraphQL. 
+
+## GraphQL Ratings API Benefits
+- Flexibility: The GraphQL Ratings API gives users the ability to access the details used to obtain the shipping rate and query what is most important to you and your business. As such, you can query 5 pieces of information or every possible field as you see fit.
+- Customization: The GraphQL Ratings API  givesusers the ability to build their own integration into ShipperHQ to retrieve shipping rates. For example, you can use it to get shipping rates into your order processing system or build a custom integration from you eCommerce cart.
+- Split Shipments: The GraphQL Ratings API enables users to leverage multi-origin fulfillment and split shipping and product groups at checkout. Retrieve tailored shipping rates and methods for each item in the order based on their shipping origin and/or product group! This way, your customer’s order does not have to be limited by the item with the longest shipping time. 
+
+## Getting Started
+ShipperHQ’s classic rating service provides rating to BigCommerce, Shopify, and Magento eCommerce platforms. The Ratings API uses GraphQL to allow for faster and more efficient API calls and PWA-compatibility.
+
+Please refer to the Introduction to GraphQL article from Graphql.org for more information about GraphQL. 
+
+## Ratings Workflow
+ShipperHQ can calculate shipping rates based on your shipping strategy. 
+
+Submit a request that includes the contents of the cart, like item’s weight, value and quantity and include destination address information 
+Can include any other shipping options required like liftgate,  residential address or specific dates for delivery
+Our API will return shipping rates for the cart using your ShipperHQ configured rules and carriers
+Depending on whether you choose to request simple rates or full shipping rates, we can return basic rate information or full details of carriers, shipments, dates and shipping options available.
+
+## Retrieving Rating Information
+You can retrieve shipping rates by querying the Rating GraphQL API. 
+
+## GraphQL API URL and Limits
+- Endpoint URL: The current GraphQL API endpoint is at https://api.shipperhq.com/v2/graphql and all requests must be ```HTTP POST``` requests with ```application/JSON``` encoded bodies.
```

Comment:
Code blocks use 3 backticks. If it's just one line, you can use 1 backtick. Can you make these changes throughout the page?

```suggestion
- Endpoint URL: The current GraphQL API endpoint is at https://api.shipperhq.com/v2/graphql and all requests must be `HTTP POST` requests with `application/JSON` encoded bodies.
```

Reply:
Good call. I did replace all of them